modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1277,6 +1296,7 @@ private void dropTableLocally(long causalityToken, String name, int tblId, List<
 
     private Set<Assignment> calculateAssignments(TableConfiguration tableCfg, int partNum) {
         return AffinityUtils.calculateAssignmentForPartition(
+                // TODO: https://issues.apache.org/jira/browse/IGNITE-19506 we must use distribution zone keys here
```

Comment:
Why did you add this TODO? Actually we will not use data nodes from zone manager here when  IGNITE-19506 will be implemented. We want to use data nodes from zone manager here when https://issues.apache.org/jira/browse/IGNITE-19425 will be implemented.